services/src/main/java/org/keycloak/services/resources/admin/IdentityProvidersResource.java

```diff
@@ -87,7 +83,7 @@ public Response getIdentityProviders(@PathParam("provider_id") String providerId
         IdentityProviderFactory providerFactory = getProviderFactorytById(providerId);
         Map<String, String> config = providerFactory.parseConfig(inputStream);
         
-        adminEvent.operation(OperationType.ACTION).resourcePath(uriInfo.getPath()).representation(config).success();
+        adminEvent.realm(realm).operation(OperationType.ACTION).resourcePath(providerFactory, uriInfo.getPath()).representation(config).success();
```

Comment:
Same here. This is a CREATE. An ACTION is something that's not creating, updating or deleting a resource. For example push not before, or send user password reset